nixos/modules/services/networking/ddclient.nix

```diff
@@ -31,7 +31,7 @@ let
   preStart = ''
     install ${configFile} /run/${RuntimeDirectory}/ddclient.conf
     ${lib.optionalString (cfg.configFile == null) (if (cfg.passwordFile != null) then ''
-      password=$(head -n 1 ${cfg.passwordFile})
+      password=$(head -n 1 ${cfg.passwordFile} | sed 's,\\,\\\\,g' | sed 's,/,\\/,g')
```

Comment:
'printf "%q"' feels like a good idea here.

Reply:
printf %q should handle all special chars, not just the slashes. Why not use it? (You resolved the conversation without comment or code change.)

Reply:
I actually added `printf`. Thank you for the suggestion :)

Reply:
Oh, sorry, I didn't notice! (I thought it'd be visible from within this threaded github view, but apparently not.)